simple_network_sim/network_of_populations.py

```diff
@@ -139,6 +208,19 @@ def distributeInfections(nodeState, newInfections):
 
 
 def doIncomingInfectionsByNode(graph, currentState, movementMultiplier):
+    """Determine the number of new infections at each node of a graph.
+
+    :param graph: A graph with each region as a node and the weights corresponding to the commutes
```

Comment:
I'd replace the word commute with movement or something else more generic. I think we are only using commute data because that's what we have, but ideally we'd use all the movements among regions.